contrib/cf-remote/cf_remote/commands.py

```diff
@@ -1,8 +1,16 @@
-from cf_remote.remote import get_info, print_info
+from cf_remote.remote import get_info, print_info, install_host
 
 
 def info(hosts, users=None):
     assert hosts
     for host in hosts:
         data = get_info(host, users)
         print_info(data)
+
+
+def install(hub, clients, bootstrap=None, package=None):
+    assert hub or clients
+    if hub:
+        install_host(hub, hub=True, package=package)
+    for host in (clients or []):
+        install_host(host, hub=False, package=package)
```

Comment:
So it's going to be the same package for hub and clients? How?

Reply:
If you specify `--package` it will be the same for all hosts, yes. Trying to do a little MVP here / minimal iterations :) The idea is that you won't have to pick the package name, it can be chosen automatically. Would you prefer if I also add `--hub-package` and `--client-package` ?

Reply:
I think it is okay if this iteration (MVP) requires the user to run the tool twice, once for the hub and once for the hosts. But I think we should avoid installing hub packages on hosts.

Reply:
Is it already done somewhere? I cannot find it.

Reply:
Added in a separate commit now.